solidity/contracts/KeepRandomBeaconOperator.sol

```diff
@@ -671,8 +675,16 @@ contract KeepRandomBeaconOperator is ReentrancyGuard {
     }
 
     /**
-     * @dev Gets the number of active groups. Expired and terminated groups are
-     * not counted as active.
+     * @notice Gets the number of active groups as currently marked in the
+     * contract. Groups marked as expired or terminated does not count as
```

Comment:
We're saying here that we return active groups and expired groups don't count as active. But in the next paragraph we state that it may include expired groups in specific cases. Maybe we could update `notice` description so we don't cause confusions?

Reply:
But we also say `... as currently marked in the contract`. 

I do agree it's confusing but I have no better idea how to express it. I am open to suggestions!

Reply:
"The number of active groups when the expired groups were last updated, without accounting for recent expirations"?

Reply:
Please see https://github.com/keep-network/keep-core/pull/1650/commits/1a28cdad5e6d6ba449490a8e673f8604d84a6a66